adam-core/src/main/scala/org/bdgenomics/adam/converters/FastqRecordConverter.scala

```diff
@@ -41,6 +41,102 @@ import scala.collection.JavaConversions._
 private[adam] class FastqRecordConverter extends Serializable with Logging {
 
   /**
+   * Parse 4 lines at a time
+   * @see parseReadPairInFastq
+   * *
+   */
+  private def parseReadInFastq(input: String,
+                               setFirstOfPair: Boolean = false,
+                               setSecondOfPair: Boolean = false,
+                               stringency: ValidationStringency = ValidationStringency.STRICT): (String, String, String) = {
+    val lines = input.split('\n')
+    require(lines.length == 4,
+      s"Input must have 4 lines (${lines.length.toString} found):\n${input}")
+
+    val readName = lines(0).drop(1)
+    if (readName.endsWith("/1") && setSecondOfPair)
```

Comment:
I have seen both, as well. I am not aware if there is a specification that lists all possibilities. I am thinking of using regex to account for all of them gradually.